src/Traits/RequestTrait.php

```diff
@@ -236,6 +236,19 @@ protected function processResponse($service, $method, $response, $resultClass)
             $this->resetLastRequest();
 
             return $result;
+        } elseif ($statusCode == StatusCode::UNAVAILABLE){
+            if ($this->ydb->needDiscovery()){
```

Comment:
No need reconnect special with unavailable code.

StatusCode::UNAVAILABLE mean - you succesfully connect to the node and the node can response to you. It mean - the node work.

I'm ok if reconnect for every error at first step (then filter some errors by list), but ydb status unavailable no need rediscovery and reconnect as special case with a lot of code.